core/templates/dev/head/player/conversation_skin_embed_directive.html

```diff
@@ -0,0 +1,716 @@
+                </span>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div ng-if="isCurrentSupplementalCardNonempty()"
+            class="conversation-skin-supplemental-card-container">
+          <img class="conversation-skin-oppia-avatar show-tutor-card"
+              ng-if="isScreenNarrowAndShowingSupplementalCard() && isCurrentSupplementalCardNonempty()"
+              ng-src="<[OPPIA_AVATAR_IMAGE_URL]>"
+              ng-click="showTutorCardIfScreenIsNarrow()"
+              alt="">
+          <div ng-if="isScreenNarrowAndShowingTutorCard()"
+              class="supplemental-mask"></div>
+          <div ng-class="{'conversation-skin-help-card help-card-standard': !isWindowTall(),
+              'conversation-skin-help-card help-card-tall': isWindowTall()}" ng-if="helpCardHtml">
+            <img class="conversation-skin-oppia-avatar" ng-src="<[OPPIA_AVATAR_IMAGE_URL]>" alt="">
+            <button type="button" class="conversation-skin-close-help-card-button" ng-click="clearHelpCard()" ng-if="!helpCardHasContinueButton">
+              <i class="material-icons md-18">&#xE5CD;</i>
+            </button>
+            <div class="conversation-skin-help-card-content" angular-html-bind="helpCardHtml"></div>
+            <br>
+            <md-button class="oppia-learner-continue-button protractor-test-continue-to-next-card-button" focus-on="<[CONTINUE_BUTTON_FOCUS_LABEL]>" ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)" ng-if="helpCardHasContinueButton" style="margin-top: 12px;" translate="I18N_PLAYER_CONTINUE_NEXT_CARD">
+            </md-button>
+          </div>
+          <div class="protractor-test-conversation-input" angular-html-bind="activeCard.interactionHtml">
+          </div>
+        </div>
+        </div>
+    </section>
+    <footer>
+        <nav class="bottom-nav row">
+            <div class="nav-left-content">
+                <div ng-if="activeCard.destStateName && activeCard.answerFeedbackPairs[activeCard.answerFeedbackPairs.length - 1].oppiaFeedbackHtml && !helpCardHasContinueButton">
+                    <md-button aria-label="" class="oppia-learner-continue-button protractor-test-continue-to-next-card-button" focus-on="<[CONTINUE_BUTTON_FOCUS_LABEL]>" ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)" translate="I18N_PLAYER_CONTINUE_BUTTON">
+                    </md-button>
+                </div>
+            </div>
+            <progress-dots  num-dots="numProgressDots" class="conversation-skin-progress-dots-in-bottom-nav">
+            </progress-dots>
+            <div class="nav-right-content">
+                <div ng-if="isOnTerminalCard()" class="logo-bottom-nav-bar">
+                    <div class="nav-bar-bottom-replay-content">
+                        <i ng-click="iframeReloadExprolation()" class="material-icons replay-nav-icon">replay</i>
+                        <p> Restart exporation?
```

Comment:
This all looks a bit cramped on the right. Maybe put the "Restart exploration" control in the bottom left? Try to make it look nice.